content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst

```diff
@@ -0,0 +1,78 @@
+Marketing activities
+====================
+
+When creating a campaign in the :guilabel:`Marketing Automation` app, the user can configure
+marketing activities. In the :guilabel:`Workflow` area, the user can click 'Add New Activity' to 
+add, edit, and customize different marketing activities to coincide with the marketing campaign.
+
+When :guilabel:`Add New Activity` is selected, a blank activity template will appear.
+
+.. image:: workflow_activities/activity-template.png
+   :align: center
+   :alt: An activity template in Odoo Marketing Automation.
+
+- :guilabel:`Activity Name`: the title of the activity.
+- :guilabel:`Activity Type`: choose between Email, Server Action (internal Odoo operation), or SMS.
+- :guilabel:`Mail Template`: choose from pre-configured templates (or create a new one on-the-fly).
+- :guilabel:`Trigger`: choose when this activity should be triggered.
+- :guilabel:`Expiry Duration`: configure to stop the actions after a specific amount of time
+  (after the scheduled date). When selected, a :guilabel:`Cancel after` field appears, in which the
+  user can choose how many :guilabel:`Hours, Days, Weeks, or Months` they want the actions to cease
+  after the initial date.
+- :guilabel:`Activity Filter`: domain related to this activity (and all subsequent Child Activities).
+- :guilabel:`Applied Filter`: activity will **only** be performed if it satisfies the specified
+  domain (filter).
```

Comment:
```suggestion
- :guilabel:`Applied Filter`: activity will *only* be performed if it satisfies the specified
  domain (filter).
```

For clarity, I changed the formatting around `only` to italics instead of bold. My reasoning is that you use bold formatting later in the doc to introduce the term `Parent Activity`, and `only` is not a term that's being introduced, it's just a word being emphasized, so it should have different formatting from `Parent Activity`.